client/src/components/settings/toggle-radio-setting.tsx

```diff
@@ -41,27 +41,41 @@ export default function ToggleRadioSetting({
           {explain ? <p id={`desc${flagName}`}>{explain}</p> : null}
         </div>
         <div className='toggle-radio-group'>
-          <label htmlFor={firstRadioId} data-checked={flag}>
-            <input
-              id={firstRadioId}
-              type='radio'
-              {...(flag && { defaultChecked: true })}
-              {...(!flag && { onChange: toggleFlag })}
-              name={flagName}
-              value='1'
-            />
-            <span>{restProps.onLabel}</span>
+          <label
+            className='outer-label'
+            htmlFor={firstRadioId}
+            data-checked={flag}
+          >
```

Comment:
You cannot nest a `label` in a `label`. If the solution requires an extra element to wrap the original `label` then it should be a `div`.